### ISSUE TITLE
uSync Empty file keeps deleting content on app startup

### ISSUE THREAD

**Issue by Simon-Ratcliffe-Epiphany** [KevinJump/uSync8#73]
This is for uSync 8.3.0 on Umbraco 8.3.

- A doctype was created and then deleted. 
- Later it was created again using the same name as the old doctype. 
- Restarting the app pool triggered an import and deleted the articles 

I have attached the uSync content type files in question. It is inherited from another so included that one too.

We have checked the permissions, and everything looks good. To get it working we manually removed the Empty file (pagetrainingarticle.config) and stripped the _vv4dxlp3 part from the "proper" config file.

[pagetrainingarticle.zip](https://github.com/KevinJump/uSync/files/4128028/pagetrainingarticle.zip)

**Comment by KevinJump** [comment]
Hi, yes, we have a fix for this now. 

We recently introduced some code to stop overwriting of config files when two things have the same name, (like in content having `home/products/MyPage` and `home/about/MyPage`. Except this was checking to egarly on things where two things sharing the same name isn't possible (like DocTypes!)

So we've moved that bit of code over to the content handlers, because its really only content and media that can do this. - Added bonus we've also fixed an issue where trashed content, wasn't storing the location it was trashed from in the config file. 

Will push a release with this out soon - just need to check we haven't made any breaking changes.
